pkg/config/default.go

```diff
@@ -106,6 +106,8 @@ func DefaultIdentityConfig() *IdentityConfig {
 		rawTokenExpiry:        DEFAULT_TOKEN_EXPIRY.String(),
 		rawTokenServerRESTAPI: "false",
 		rawDeleteInstanceID:   "true",
+		rawShutdownTimeout:    "5s",
+		rawShutdownDelay:      "0s",
```

Comment:
```suggestion
		rawShutdownTimeout:    DEFAULT_SHUTDOWN_TIMEOUT.String(),
		rawShutdownDelay:      DEFAULT_SHUTDOWN_DELAY.String(),

		ShutdownTimeout:      DEFAULT_SHUTDOWN_TIMEOUT,
		ShutdownDelay:      DEFAULT_SHUTDOWN_DELAY,
```
please add `DEFAULT_SHUTDOWN_TIMEOUT` & `DEFAULT_SHUTDOWN_DELAY` like `DEFAULT_TOKEN_EXPIRY`.

Reply:
Can we have the same order of fields in the `model.go` and `default.go`?🙈

- https://github.com/AthenZ/k8s-athenz-sia/blob/de8ecdcee1ca72ad98307a2294d7765b70e4c4fc/pkg/config/model.go#L25-L60
- https://github.com/AthenZ/k8s-athenz-sia/blob/de8ecdcee1ca72ad98307a2294d7765b70e4c4fc/pkg/config/default.go#L66-L104